src/targets/gpu/CMakeLists.txt

```diff
@@ -252,6 +254,9 @@ endif()
 message(STATUS "clang-offload-bundler: ${MIGRAPHX_OFFLOADBUNDLER_BIN}")
 message(STATUS "extractkernel: ${MIGRAPHX_EXTRACT_KERNEL}")
 
+target_compile_definitions(migraphx_gpu PRIVATE "-DMIGRAPHX_MLIR_MIOPEN_SUPPORT")
+target_link_libraries(migraphx_gpu PUBLIC MLIRMIOpenThin)
```

Comment:
This should search for the library with `find_library`, similar to how MIOpen finds MLIR:

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/CMakeLists.txt#L318